solr/core/src/test/org/apache/solr/update/processor/ParsingFieldUpdateProcessorsTest.java

```diff
@@ -992,6 +992,14 @@ public void testLenient() throws IOException {
     assertParsedDate("Friday Oct 7 13:14:15 2005", Date.from(inst20051007131415()), "parse-date-patterns-default-config");
   }
 
+  public void testRfc2616() throws Exception {
+    assertParsedDate("Fri Oct 7 13:14:15 2005" , Date.from(inst20051007131415()), "parse-date-patterns-default-config");
+  }
+
+  public void testRfc2616Leniency() throws Exception {
```

Comment:
Should be named testAsctimeLeniency (see my previous comment).  Glad to see this test works.

'course there  are already some tests with similar names... so feel free to combine some.  For example the "testLenient" method is an Ansi C test of leniency; by all means combine tests into one test method as appropriate.